### PR TITLE
Enabled support for push notification on android with FCM

### PR DESCRIPTION
**Pull request by jasondellaluce**
Related to [https://github.com/nedrocks/delphis_app/pull/49](https://github.com/nedrocks/delphis_app/pull/49) and addresses [https://github.com/nedrocks/delphis_app/issues/46](https://github.com/nedrocks/delphis_app/issues/46). Enables push notification sending on Android platfroms, using Ably's APIs.

### CODE REVIEW

File: internal/notif/ably_notif.go [hunk 1]
```diff
@@ -24,27 +26,53 @@ type PushNotificationBody struct {
...
-		Recipient: PushNotificationRecipient{
-			TransportType: "apns",
-			DeviceToken:   *device.Token,
-		},
-		Notification: pushBody,
-	})
+
+	var body []byte = nil
+	var err error = nil
+	if strings.ToLower(device.Platform) == "android" {
```
Comment (nedrocks) [inline internal/notif/ably_notif.go:56]: Should we do a verification that `device.Token` is not `nil`?

Comment (jasondellaluce) [reply internal/notif/ably_notif.go:56]: Yeah that’s true. I’ll add it in the next commit.

File: internal/notif/ably_notif.go [hunk 2]
```diff
@@ -24,27 +26,57 @@ type PushNotificationBody struct {
...
+	} else if strings.ToLower(device.Platform) == "ios" {
+		body, err = json.Marshal(ApplePushNotification{
+			Recipient: ApnsPushNotificationRecipient{
+				TransportType: "apns",
+				DeviceToken:   *device.Token,
+			},
+			Notification: pushBody,
+		})
+	} else {
+		return false, errors.New("Unknown platform for device: " + device.Platform)
```
Comment (nedrocks) [inline internal/notif/ably_notif.go:77]: nit: Generally it's preferable to use `fmt.Errorf("blah blah %s", device.Platform)

Comment (jasondellaluce) [reply internal/notif/ably_notif.go:77]: Ok. In this case, do we still return the error tho?

**Pull request merged**